chain/client/src/client.rs

```diff
@@ -962,6 +988,18 @@ impl Client {
         Ok(())
     }
 
+    /// Gets the advanced timestamp delta in nanoseconds for sandbox once it has been fast-forwarded
+    #[cfg(feature = "sandbox")]
+    pub fn sandbox_delta_time(
+        &self,
+        delta_height: near_primitives::types::BlockHeightDelta,
+    ) -> u64 {
+        let avg_block_prod_time = (self.config.min_block_production_delay.as_nanos()
+            + self.config.max_block_production_delay.as_nanos())
+            / 2;
+        (delta_height as u128 * avg_block_prod_time) as u64
```

Comment:
Theoratically it's better to check overflow and panic. but I guess it's fine as u64 can hold sufficiently big number of nano seconds.

Reply:
Converted this to a `try_into().expect()` to panic if necessary